charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -34,25 +32,17 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | Parameter                         | Description                          | Default                                   |
 | --------------------------------- | ------------------------------------ | ----------------------------------------- |
 | `master.initScripts`              | List of Jenkins init scripts         | `[]`                                      |
-| `master.overwriteConfig`          | Replace init scripts and config w/ ConfigMap on boot  | `false`                  |
-| `master.credentialsXmlSecret`     | Kubernetes secret that contains a 'credentials.xml' file | Not set               |
-| `master.secretsFilesSecret`       | Kubernetes secret that contains 'secrets' files | Not set                        |
```

Comment:
I think we need to keep this. it's used, for example, to set `hudson.util.Secret` and `master.key` from a Kubernetes Secret so credentials are portable between instances. this isn't supported by JCasC yet...see [here](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/secrets.adoc#passing-credentials-as-encrypted-text).

Reply:
I would rather point people to https://jenkinsci.github.io/kubernetes-credentials-provider-plugin/.

Reply:
`kubernetes-credentials-provider-plugin` is great for loading top-level credentials from k8s secrets...and it appears it will support folder-level credentials once [this PR](https://github.com/jenkinsci/kubernetes-credentials-provider-plugin/pull/40) is merged...but I don't expect it will ever support loading the encryption keys

Reply:
Why do you want to load the encryption key?

Reply:
blue/green deployments require encrypted folder-level credentials to be portable

I'm realizing I won't need to load the encryption key once `kubernetes-credentials-provider-plugin` supports folder-level credentials...but it's needed in the meantime to manage encrypted folder-level credentials via the Job DSL plugin